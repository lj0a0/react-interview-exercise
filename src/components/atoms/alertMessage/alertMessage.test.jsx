import React from "react";
import AlertMessage from "./alertMessage";
import renderer from "react-test-renderer";

test("AlertMessage snapshot 0", () => {
  const component = renderer.create(<AlertMessage error={{ code: 0 }} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("AlertMessage snapshot 1", () => {
  const component = renderer.create(
    <AlertMessage error={{ code: 1, message: "" }} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("AlertMessage snapshot 2", () => {
  const component = renderer.create(
    <AlertMessage error={{ code: 2, message: "" }} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("AlertMessage snapshot 3", () => {
  const component = renderer.create(
    <AlertMessage error={{ code: 3, message: "" }} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("AlertMessage snapshot 4", () => {
  const component = renderer.create(
    <AlertMessage error={{ code: 4, message: "" }} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
