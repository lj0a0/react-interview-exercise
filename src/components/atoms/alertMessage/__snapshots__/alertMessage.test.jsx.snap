// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertMessage snapshot 0 1`] = `null`;

exports[`AlertMessage snapshot 1 1`] = `
<div
  className="alert alert-success"
  id="alert_success"
  role="alert"
>
  
</div>
`;

exports[`AlertMessage snapshot 2 1`] = `
<div
  className="alert alert-danger"
  id="alert_danger"
  role="alert"
>
  
</div>
`;

exports[`AlertMessage snapshot 3 1`] = `
<div
  className="alert alert-warning"
  id="alert_warning"
  role="alert"
>
  
</div>
`;

exports[`AlertMessage snapshot 4 1`] = `
<div
  className="alert alert-info"
  id="alert_info"
  role="alert"
>
  
</div>
`;
